### PR TITLE
13 rental form component

### PR DESCRIPTION
**Pull request by rawand1998**
form

### CODE REVIEW

**Review by aaamra — APPROVED** [review]

File: client/src/Components/RentalForm/index.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    handleError(async () => {
+      try {
+        await axios.post('/api/v1/rentals/1', {
+          location: data.location,
+          pickUpDateTime: data.pickUpDateTime,
+          pickOffDateTime: data.pickOffDateTime,
+        });
+        setOpen(false);
+      } catch (err) {
+        console.log(err);
```
Comment (LinaYahya) [inline client/src/Components/RentalForm/index.js:49]: use alert instead

File: client/src/Components/RentalForm/index.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        location: false,
+      });
+    }
+    callback();
+  };
+
+  const rental = () => {
+    handleError(async () => {
+      try {
+        await axios.post('/api/v1/rentals/1', {
```
Comment (LinaYahya) [inline client/src/Components/RentalForm/index.js:42]:
```suggestion
        await axios.post(`/api/v1/rentals/${id}`, {
```
get id from props

File: client/src/Components/RentalForm/index.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+          widthInput="1.25"
+          onChange={(e) => {
+            setData({ ...data, location: e.target.value });
+          }}
+          error={error.location}
+          helperText={error.location ? 'This Field is required' : ''}
+        />
+
+        {' '}
+        <br />
```
Comment (LinaYahya) [inline client/src/Components/RentalForm/index.js:69]: try not to use

File: client/src/Components/Confirm/inedx.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <div>
+      <Button loading={false} handelClick={handleClickOpen} text="Rent Now" />
```
Comment (LinaYahya) [inline client/src/Components/Confirm/inedx.js:24]: remove button to the form itself

File: client/src/Components/RentalForm/index.js [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+      });
+    } else {
+      setError({
+        location: false,
+      });
+    }
+    callback();
+  };
+
+  const rental = () => {
```
Comment (LinaYahya) [inline client/src/Components/RentalForm/index.js:39]: rename your function to rentCar

File: client/src/Components/RentalForm/index.js [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+          setOpen={setOpen}
+          className="confirm-btn"
+        />
+      </form>
+    </div>
+  );
+}
+RentalForm.propTypes = {
+//   carName: PropTypes.string.isRequired,
+//   id: PropTypes.number.isRequired,
```
Comment (LinaYahya) [inline client/src/Components/RentalForm/index.js:110]: delete those lines

File: client/src/Components/RentalForm/style.css [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding-top: 2.5rem;
+    padding-bottom: 2rem;
+    align-items: center;
+    
+    
+}
+.container{
```
Comment (LinaYahya) [inline client/src/Components/RentalForm/style.css:11]: rename your className

File: server/controllers/rentals/addRentalController.js [hunk 8]
```diff
@@ -33,6 +33,7 @@ const addRentalController = async (req, res, next) => {
 
     return res.status(201).json({ message: 'Car rented successfully' });
   } catch (err) {
+    console.log(err);
```
Comment (LinaYahya) [inline server/controllers/rentals/addRentalController.js:36]:
```suggestion
```

**Review by LinaYahya — CHANGES_REQUESTED** [review]

File: client/src/Components/RentalForm/index.js (hunk 3 above)
Comment (rawand1998) [reply client/src/Components/RentalForm/index.js:69]: from eslint

File: client/src/Components/RentalForm/index.js (hunk 5 above)
Comment (rawand1998) [reply client/src/Components/RentalForm/index.js]: done

File: client/src/Components/RentalForm/index.js (hunk 6 above)
Comment (rawand1998) [reply client/src/Components/RentalForm/index.js]: done

File: client/src/Components/RentalForm/style.css (hunk 7 above)
Comment (rawand1998) [reply client/src/Components/RentalForm/style.css]: done

File: client/src/Components/RentalForm/index.js (hunk 2 above)
Comment (rawand1998) [reply client/src/Components/RentalForm/index.js:42]: done

File: client/src/Components/RentalForm/index.js (hunk 1 above)
Comment (rawand1998) [reply client/src/Components/RentalForm/index.js:49]: done

**Review by AhemdAboAbadi — APPROVED** [review]

File: server/controllers/rentals/addRentalController.js (hunk 8 above)
Comment (rawand1998) [reply server/controllers/rentals/addRentalController.js:36]: done

**Review by asmaaamin — APPROVED** [review]

**Pull request merged**